packages/vue/src/components/authenticator.vue

```diff
@@ -522,10 +481,10 @@ const hasRouteComponent = computed(() => {
     </div>
   </div>
   <slot
-    v-if="state?.matches('authenticated')"
-    :user="user"
-    :state="state"
-    :signOut="signOut"
-    :send="send"
+    v-if="route === 'authenticated'"
+    :user="(user as any)"
+    :state="(state as any)"
+    :signOut="(signOut as any)"
+    :send="(send as any)"
```

Comment:
Added for backwards compatibility and to avoid breaking changes.

From my experience my vscode could not infer the type through slots, but wanted to be sure.

Reply:
Can we add a comment? Know that there is a good chance this code might just disappear during a refactor, but in case it still exists in the future 😅

Reply:
Done in 9fec28ccd842ead311f17a2a9de0fe6ea78b9e1f 🙏